velox/connectors/hive/storage_adapters/hdfs/HdfsWriteFile.h

```diff
@@ -15,11 +15,14 @@
  */
 #pragma once
 
-#include <hdfs/hdfs.h>
 #include "velox/common/file/File.h"
+#include "velox/external/hdfs/ArrowHdfsInternal.h"
```

Comment:
remove headers and forward declare
```
namespace filesystems::arrow::io::internal {
  class LibHdfsShim;
}
```

Reply:
@majetideepak https://github.com/JkSelf/velox/blob/libhdfs/velox/connectors/hive/storage_adapters/hdfs/HdfsWriteFile.h#L40 need the LibHdfsShim.

Reply:
Its a pointer, you can forward declare.